diff --git a/src/rules/nx-plugin-checks.js b/src/rules/nx-plugin-checks.js
index a0e3cd4bafb367db5a6b10ac2490c54ab6627fe9..f876aea5ba89e7252127f11926d87d52e453bc25 100644
--- a/src/rules/nx-plugin-checks.js
+++ b/src/rules/nx-plugin-checks.js
@@ -73,7 +73,7 @@ exports.default = utils_1.ESLintUtils.RuleCreator(() => ``)({
     defaultOptions: [DEFAULT_OPTIONS],
     create(context) {
         // jsonc-eslint-parser adds this property to parserServices where appropriate
-        if (!context.parserServices.isJSON) {
+        if (!context.sourceCode.parserServices.isJSON) {
             return {};
         }
         const { projectGraph, projectRootMappings } = (0, project_graph_utils_1.readProjectGraph)(exports.RULE_NAME);
